### ISSUE TITLE
[Desktop] bubbles do not show properly on initial display

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#11838]
## Test plan
Issue should be fixed as part of Chromium 87
- Check https://bugs.chromium.org/p/chromium/issues/detail?id=1112244#c19
- Milestone or notes should indicate this was merged to Chromium 87. If not, we can move this issue to milestone 1.19 (Chromium 88)

If everything looks good, test steps are below under `Steps to Reproduce`

## Description 

Modals are not displaying properly on initial load, however if you close and reopen the modal does show as expected.

Example:
When you navigate to a site in a different language and you see the offer to translate popup (the one to install Google Translate), the popup does not display correctly on initial load. If you click away and then reopen the popup it does display correctly. 

Example 2:
Password modals show oddly as well. If you close/reopen by clicking on the "key" icon it shows as expected.

Does not reproduce with 1.14.x (C85) version.

## Steps to Reproduce

Scenario 1:
   1. Clean profile 1.15.x
   2. Navigate to a site in a different language (ex. if your OS is set to English, try interia.pl or 3zsistemi.si)
   3. You get the offer to translate popup but the colors/styling is incorrect

## Actual result:

Dark:
<img width="1194" alt="1 15 x - initial" src="https://user-images.githubusercontent.com/28145373/94157409-3697a200-fe4f-11ea-98b0-cc0dddd5342b.png">

Light:
<img width="1192" alt="light" src="https://user-images.githubusercontent.com/28145373/94157425-3bf4ec80-fe4f-11ea-9103-0ab70b3de256.png">

## Expected result:
Should show as it does in 1.14.x which is also how it shows if you click away and then reopen this popup manually from the URL bar:
Dark
<img width="1194" alt="reload dark" src="https://user-images.githubusercontent.com/28145373/94157490-4e6f2600-fe4f-11ea-9239-b52250e88c97.png">

Light
<img width="1193" alt="reload light" src="https://user-images.githubusercontent.com/28145373/94157493-4fa05300-fe4f-11ea-8f80-b5331a3be453.png">

Scenario 2:
   1. Clean profile 1.15.x
   2. Navigate to a site where you need to login.
   3. Enter login info/login
   3. You get the offer to save your password but the colors/styling is incorrect

## Actual result:

Dark:
<img width="359" alt="Screen Shot 2020-09-24 at 10 46 46 AM" src="https://user-images.githubusercontent.com/28145373/94161505-bfb0d800-fe53-11ea-8d7d-74bcea9a1e65.png">

Light:
<img width="371" alt="Screen Shot 2020-09-22 at 11 18 59 PM" src="https://user-images.githubusercontent.com/28145373/94161557-ce978a80-fe53-11ea-8fc2-de533c2ee908.png">

## Expected result:
Should show as it does in 1.14.x which is also how it shows if you close and then reopen this popup manually from the URL bar:
<img width="339" alt="Screen Shot 2020-09-24 at 10 51 04 AM" src="https://user-images.githubusercontent.com/28145373/94161685-f2f36700-fe53-11ea-8396-86908cbcd9a5.png">

## Reproduces how often: 

easily

## Brave version (brave://version info)

Brave | 1.15.60 Chromium: 86.0.4240.42 (Official Build) dev (x86_64)
-- | --
Revision | 418ccfc2b69112bd79d5ffb38b533642e64e6af8-refs/branch-heads/4240@{#773}
OS | macOS Version 10.14.6 (Build 18G3020)

Brave | 1.16.35 Chromium: 86.0.4240.42 (Official Build) nightly (x86_64)
-- | --
Revision | 418ccfc2b69112bd79d5ffb38b533642e64e6af8-refs/branch-heads/4240@{#773}
OS | macOS Version 10.14.6 (Build 18G3020)

## Version/Channel Information:

- Can you reproduce this issue with the current release? no using 1.14.x C85
- Can you reproduce this issue with the beta channel? yes using 1.15.x C86
- Can you reproduce this issue with the nightly channel? yes using 1.16.x C86

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

cc @mkarolin @rebron @bsclifton

**Comment by btlechowski** [comment]
Not reproducible on Linux

Brave | 1.15.60 Chromium: 86.0.4240.42 (Official Build) dev (64-bit)
-- | --
Revision | 418ccfc2b69112bd79d5ffb38b533642e64e6af8-refs/branch-heads/4240@{#773}
OS | Ubuntu 18.04 LTS

![image](https://user-images.githubusercontent.com/34715963/94158242-76c44800-fe82-11ea-8ddf-c7bd782322c7.png)
![image](https://user-images.githubusercontent.com/34715963/94158507-c0ad2e00-fe82-11ea-8f21-d2f21d300010.png)

Not reproducible on Windows


Brave | 1.15.60 Chromium: 86.0.4240.42 (Official Build) dev (64-bit)
-- | --
Revision | 418ccfc2b69112bd79d5ffb38b533642e64e6af8-refs/branch-heads/4240@{#773}
OS | Windows 7 Service Pack 1 (Build 7601.24544)



![image](https://user-images.githubusercontent.com/34715963/94158667-f7834400-fe82-11ea-8f45-326648871661.png)

![image](https://user-images.githubusercontent.com/34715963/94158862-2bf70000-fe83-11ea-89c0-8c0b4bd8b831.png)

**Comment by kjozwiak** [comment]
Reproduced the issue with the `Save Password` modal as well. Example:

<img width="371" alt="Screen Shot 2020-09-22 at 11 18 59 PM" src="https://user-images.githubusercontent.com/2602313/94161276-82e4e100-fe53-11ea-91c3-54adb268d070.png">

Used the following build to reproduce:

```
Brave | 1.15.56 Chromium: 85.0.4183.102 (Official Build) dev (64-bit)
-- | --
Revision | ffe848af6a5df4fa127e2929331116b7f9f1cb30-refs/branch-heads/4183@{#1770}
OS | macOS Version 10.15.6 (Build 19G73)
```

**Comment by btlechowski** [comment]
Not reproducible with Passwords modal on Linux

Brave | 1.15.60 Chromium: 86.0.4240.42 (Official Build) dev (64-bit)
-- | --
Revision | 418ccfc2b69112bd79d5ffb38b533642e64e6af8-refs/branch-heads/4240@{#773}
OS | Ubuntu 18.04 LTS

![image](https://user-images.githubusercontent.com/34715963/94161818-63b37700-fe86-11ea-891f-5d309c77485f.png)

**Comment by bsclifton** [comment]
Assigned @petemill who is checking this out now 😄

**Comment by petemill** [comment]
This is an upstream bug https://bugs.chromium.org/p/chromium/issues/detail?id=1112244#c3

**Comment by petemill** [comment]
Note STR in upstream bug around the inactive colors of the "reset" button in the zoom dialog after pressing cmd-+

**Comment by LaurenWags** [comment]
Note - issue for light them was broken out into a separate issue per today's discussion: https://github.com/brave/brave-browser/issues/11916

**Comment by bsclifton** [comment]
Issue has been fixed upstream in Chromium 88; they're working on uplift to Chromium 87
https://bugs.chromium.org/p/chromium/issues/detail?id=1112244#c19

We should be able to check this with 1.17 if the patch is accepted 😄

**Comment by LaurenWags** [comment]
labelling as `QA/Blocked` until patch mentioned in https://github.com/brave/brave-browser/issues/11838#issuecomment-722535169 is included in 1.17.x.

**Comment by rebron** [comment]
Verified and looks good on:

Brave | 1.17.69 Chromium: 87.0.4280.60 (Official Build) (x86_64)
-- | --
Revision | 12697cfeb273d7de95cf9b18350d2c457f58224c-refs/branch-heads/4280@{#1352}
OS | macOS Version 11.0.1 (Build 20B29)

![Screen Shot 2020-11-16 at 11 01 43 AM](https://user-images.githubusercontent.com/5951041/99296407-94b38680-27fb-11eb-937c-0edccfea3646.png)
![Screen Shot 2020-11-16 at 11 01 23 AM](https://user-images.githubusercontent.com/5951041/99296421-9aa96780-27fb-11eb-833c-58d48242193d.png)
